packages/nc-gui-v2/components/cell/DatePicker.vue

```diff
@@ -1,91 +1,63 @@
 <script setup lang="ts">
 import dayjs from 'dayjs'
-import { computed } from '#imports'
+import customParseFormat from 'dayjs/plugin/customParseFormat'
+
+import { ColumnInj, ReadonlyInj } from '~/context'
+const { modelValue } = defineProps<Props>()
+
+const emit = defineEmits(['update:modelValue'])
+
+dayjs.extend(customParseFormat)
```

Comment:
Should be in date utils file - all dayjs plugins should be installed there and not all over the app :)